test/inputs/missing-location/constraint-code.ts

```diff
@@ -0,0 +1,3 @@
+  export default () => {
+    return Real.Resource("state of charge").lessThan(0.3).split(0)
+  }
```

Comment:
I don't think this needs to exist because you hard-coded it in the UserCodeRunner